### DISCUSSION TITLE
Dont get the "power correction factor"

### DISCUSSION THREAD

**Discussion by Fenar91** [doudar/SmartSpin2k#422] [Q&A]
Hi Guys,
im using a one sided Assioma-pedal and currently tried the myhomefit-App. To use the app, i had to connect my powermeter with my ss2k via BLE-Scanner and then the ss2k with the app. 
But the power was unrealisticly low, so i have to correct it. But how do i know, which factor is perfect? Why does ss2k just dont take the power the pedal is creating? 

Greeting

**Comment by eMadman** [comment]
With the assioma, you should not have to adjust the power correction factor.  Are you sure it's correctly configured in the Assioma app?  It's possible that Myhomefit isn't reading it properly.  Does the log from SS2K display the expected wattage?  How about when you have the pedals connected to a bike computer or directly to Zwift?

There's reports of this type of issue with these all over the place: https://www.reddit.com/r/cycling/comments/k3j0iw/incorrect_power_from_single_pedal_power_meter/

**Reply by Fenar91** [reply]
On Zwift is the power very low too compared with the direct connection without ss2k ble Scanner. I got with the direct connection nearly the double power, which is realistic. Maybe because the powermeter is onesided?

**Reply by doudar** [reply]
Yes, its because the BLE specs say single sided power meter pedals should report only their power. Which leaves the app (or headunit) to double the power where needed.

Favero has a double power output setting in their configuration app for the pedals. That would be my recommendation. 

We stick with ignoring the single/dual reporting in the BLE spec only because most other apps do as well. If it becomes consistent that apps start using the single sided bit in the bit field, we will start using it as well.

**Comment by doudar** [comment]
> Hi Guys,
> 
> im using a one sided Assioma-pedal and currently tried the myhomefit-App. To use the app, i had to connect my powermeter with my ss2k via BLE-Scanner and then the ss2k with the app. 
> 
> But the power was unrealisticly low, so i have to correct it. But how do i know, which factor is perfect? Why does ss2k just dont take the power the pedal is creating? 
> 
> 
> 
> Greeting

As @eMadman stated above, SS2K is only reporting the power it receives. Assioma is very accurate so it will either be correct or 1/2 of the expected power. 

First check your pedal settings in the Favero app and make sure your pedal is setup for single sided use.

If this doesn't fix it, then you can either double the power on the pedal in the Favero app or select 2 for the power correction factor in the SS2K settings.

**Reply by Fenar91** [reply]
U guys so smart, really, thank u!

**Reply by doudar** [reply]
Thanks @Fenar91 , we're always happy to help!

**Reply by Fenar91** [reply]
Hi Guys,
it is true, that my devide does need a direct lan connection with a cable to my network, so i can enter the settings in ss2k.local, right? Why i can not just go into the settings (ss2k.local) with my Phone via wlan after i connect the SS2k to my network?

Greetings.

**Reply by MarkusSchneider** [reply]
Sounds like an DNS problem. Can you connect SS2K by using its IP-Address via phone?
Had the same problem an I solved it with a static IP in the DHCP Server configuration of the router.
Do you have more then one wireless lan access point?

**Reply by doudar** [reply]
It really depends on how your network/router and the device you're using handles the MDNS protocol. Some work with it, others do not.

**Reply by Fenar91** [reply]
Hi Guys, 
thank u for the reply. I only enter the settings with my pc which is local connected with a cable. I try the DCP-Settings like u mentioned.

Thank u very much!
